.github/workflows/update-containerd.yaml

```
@@ -29,9 +29,15 @@ jobs:
         labels: |
           automated-pr
           containerd
+          type::chore
         draft: false
         base: "main"
-        body: "Automated changes by the [cron-containerd-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-containerd.yaml) GitHub action"
+        body: |
+          Automated changes by the [cron-containerd-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-containerd.yaml) GitHub action
+          
+          ```release-note
+            Added [Containerd](https://kurl.sh/docs/add-ons/containerd) ${{ steps.update.outputs.containerd_version }}
```

Can you follow the pattern we use in release notes?

```suggestion
            Adds [Containerd add-on](https://kurl.sh/docs/add-ons/containerd) version(s) ${{ steps.update.outputs.containerd_version }}.

---

For continerd can you change output for containerd_version to:

```
    echo "::set-output name=containerd_version::${VERSIONS[*]}"
```